src/Microsoft.Health.Dicom.Core.UnitTests/Features/Common/DicomFileNameWithPrefixTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Dicom.Core.Features.Common;
+using Microsoft.Health.Dicom.Core.Features.Model;
+using Microsoft.Health.Dicom.Tests.Common;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Core.UnitTests.Features.Common;
```

Comment:
It makes sense to me for these files to be in the Blob project instead of Core, they are implementation details of blob storage.

Reply:
Moved